index.html

```diff
@@ -16,117 +16,133 @@
-          <ul class="languages-work">
-            <li class="html-box"><p class="code-name">html</li>
-            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
-            <li class="ruby-box"><p class="code-name">Ruby</p></li>
-          </ul>
-          
-       </div>
-          
-       <div class="worktwo">
+      <div class="worktwo two-disktop">
+        <div class="work-box-two">
         <h5 class="work-two-title">Profesional Art Printing Data</h5>
         <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
           or sign-ups required. has been the industry's standard</p>
-          <div class="see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
           <ul class="languages-work">
             <li class="html-box"><p class="code-name">html</li>
             <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
             <li class="ruby-box"><p class="code-name">Ruby</p></li>
           </ul>
-          
-       </div>
-       <div class="worktwo">
-        <h5 class="work-two-title">Profesional Art Printing Data</h5>
-        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
-          or sign-ups required. has been the industry's standard</p>
+        </div>
           <div class="see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
-          <ul class="languages-work">
-            <li class="html-box"><p class="code-name">html</li>
-            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
-            <li class="ruby-box"><p class="code-name">Ruby</p></li>
-          </ul>
-          
        </div>
-       <div class="worktwo">
-        <h5 class="work-two-title">Profesional Art Printing Data</h5>
-        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
-          or sign-ups required. has been the industry's standard</p>
-          <div class="see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
-          <ul class="languages-work">
-            <li class="html-box"><p class="code-name">html</li>
-            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
-            <li class="ruby-box"><p class="code-name">Ruby</p></li>
-          </ul>
           
-       </div>
-       <div class="worktwo">
-        <h5 class="work-two-title">Profesional Art Printing Data</h5>
-        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
-          or sign-ups required. has been the industry's standard</p>
-          <div class="see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
+       <div class="workthree worktwo">
+        <div class="work-box-three work-box-two">
+        <h5 class="work-three-title work-two-title">Data Dashboard Healthcare</h5>
```

Comment:
[ ] kindly add a hover effect to your cards, so when we hover on with the mouse the description and dark opacity disappear and a See Project button appears at the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository.👀

Reply:
hello Nedjwab the dark opacity is engraved in the card image and in the project requirements nothing of that sort is written in it  https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md

Reply:
you can check the link

Reply:
@Cluab , it's showing in the design, this is the hover effect :
![hoverdesign](https://user-images.githubusercontent.com/62156607/193107248-4956b0a0-1e38-4aa0-bafd-21a3c8b17674.PNG)
your card without the hover effect should look like this : 

![without](https://user-images.githubusercontent.com/62156607/193107587-7f6219c1-66d4-40a0-9dba-d05f01c784af.PNG)